### DISCUSSION TITLE
allow any channel to run a campaign

### DISCUSSION THREAD

**Discussion by mrusteberg** [chatwoot/chatwoot#3721] [Feature Requests]
right now, campaigns are limited to SMS for 'one off' campaigns and livechat for 'ongoing' campaigns.

for an omni-channel customer engagement platform, the expectation would be to be able to send campaigns out through any channel that supports outbound messaging.

additionally, like others have said, setting automation rules to trigger messages based on certain events will be making this fully complete. 

i'd be happy to sponsor this development, particularly for email.

**Comment by sojan-official** [comment]
@mrusteberg It's on our roadmap to include support across inboxes. But the way campaigns can behave in different channels is much different compared to one - one message.  Also, many channels don't give you the APIs for bulk messaging.  But please do watch our roadmap for campaign support for additional channels.

**Comment by mrusteberg** [comment]
don't see it in the icebox: https://github.com/chatwoot/chatwoot/milestone/2

**Comment by da411d** [comment]
How about Telegram campaigns?
